src/OpenSage.Game/Logic/Object/Update/ProductionExit/QueueProductionExitUpdate.cs

```diff
@@ -16,7 +16,7 @@ internal QueueProductionExitUpdate(QueueProductionExitUpdateModuleData moduleDat
 
         Vector3? IProductionExit.GetNaturalRallyPoint() => _moduleData.NaturalRallyPoint;
 
-        public int ExitDelay => _moduleData.ExitDelay; 
+        public int ExitDelay => _moduleData.ExitDelay > 0 ? _moduleData.ExitDelay : 300; 
```

Comment:
If this is the default value, I think it should be on the property declaration [here](https://github.com/OpenSAGE/OpenSAGE/blob/aabedaf9537749f9e522f8cf97f5ad3085aca52f/src/OpenSage.Game/Logic/Object/Update/ProductionExit/QueueProductionExitUpdate.cs#L48).

Reply:
fixed

Reply:
just checked, the corresponding building explicitly sets it to 0, so i removed the default value of 300 alltogether